NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
```

Comment:
Yes if you simply remove the `else`, then your function no longer returns anything, thus the `undefined`. You've got a function, `add`, that already implements the `x + y` behaviour. so instead of duplicating that behaviour here, just invoke the function you already have.

```
if (...) {
  alert(...);
} else {
  return add(x,y);
}
```